constructors.js

```diff
@@ -108,3 +138,48 @@
    * @param  {Spellcaster} target         The spell target to be inflicted.
    * @return {boolean}                    Whether the spell was successfully cast.
    */
+Spellcaster.prototype.invoke = function(spell, target){
+
+  //if spell = Spell OR spell = DamageSpell
+  if(spell instanceof Spell || spell instanceof DamageSpell){
```

Comment:
This works but as an exercise, can you think of a way to write this so that it's shorter?